src/config/merge.rs

```diff
@@ -391,4 +418,24 @@ mod tests {
             BTreeMap::from([("a", Acc(4)), ("b", Acc(2)), ("c", Acc(5))])
         );
     }
+
+    #[test]
+    fn test_chainable_merge() {
```

Comment:
No need to keep the same test case as both a regular unit test and a doctest

Reply:
Here is a usecase: https://github.com/stackabletech/hdfs-operator/pull/164/files#diff-b10253ea28ef51076309f375e97cd5e2795500b0e07c5326c413055bffcb4a85R253

I will remove the test